src/marketing/page-builder-add-product-recs.md

```diff
@@ -5,6 +5,9 @@ title: Add Product Recommendations
 
 Use the Product Recommendations content type to add an existing, active [recommendation unit]({% link marketing/create-new-rec.md %}) to a page, block, or dynamic block.
 
+{:.bs-callout-info}
+Product Recommendations are compatible with Page Builder starting in Magento 2.3.0, but not supported.
```

Comment:
this makes it sound like PB is not supported "starting in Magento 2.3.0". can we clarify that it's only that version, and the rest are supported?

Reply:
```suggestion
The Product Recommendations content type in Page Builder is supported in Magento 2.3.1 and later. In Magento 2.3.0, is compatible but not supported.
```